packages/croncat-sdk-tasks/src/types.rs

```diff
@@ -292,13 +292,8 @@ pub struct CroncatQuery {
     /// This is address of the queried module contract.
     /// For the addr can use one of our croncat-mod-* contracts, or custom contracts
     ///
-    /// One requirement for custom contracts: query return value should be formatted as:
-    /// ```
-    /// {
-    ///     result: bool,
-    ///     data: Binary,
-    /// }
-    /// ```
+    /// One requirement for custom contracts: query return value should be formatted as a:
+    /// [`QueryResponse`]: mod_sdk::types::QueryResponse
```

Comment:
Nice, I like it way more, but should be 
```suggestion
    /// [`mod_sdk::types::QueryResponse`]
```